src/main/java/net/mcreator/ui/modgui/ToolGUI.java

```diff
@@ -210,8 +210,8 @@ public ToolGUI(MCreator mcreator, ModElement modElement, boolean editingMode) {
 		harvestLevel.setOpaque(false);
 		efficiency.setOpaque(false);
 
-		selp.add(HelpUtils
-				.wrapWithHelpButton(this.withEntry("common/gui_name"), L10N.label("elementgui.common.name_in_gui")));
+		selp.add(HelpUtils.wrapWithHelpButton(this.withEntry("common/gui_name"),
```

Comment:
Why did you refactor lines like this again? Please revert all occurrences as to limit the number of changes in a single PR to only the necessary ones that are relevant to the issue being fixed/done.

Reply:
Also breaking lines like this will be reverted when I run code cleanup.